packages/esm-billing-app/src/claims/dashboard/claims-dashboard.scss

```
@@ -0,0 +1,27 @@
+@use '@carbon/colors';
+@use '@carbon/layout';
+@use '@carbon/type';
+
+.container {
+  height: calc(100vh - 3rem);
+}
+
+.claimsTableContainer {
+  margin: 2rem 1rem 1rem 1rem;
```

since we already import `@carbon/layout`  we can use it here
```suggestion
  margin: layout.$$spacing-07 layout.$$spacing-05 layout.$$spacing-05 layout.$$spacing-05;
```